includes/ajax-functions.php

```diff
@@ -1437,39 +1437,47 @@ function edd_admin_order_get_item_amounts() {
 	// There is not currently API support for `OrderItem`-level `OrderAdjustment`s.
 	$adjustments = array();
 
+	global $edd_flat_discount_total;
+
 	foreach ( $discounts as $discount_id ) {
+		$edd_flat_discount_total = 0;
+
 		$d = edd_get_discount( $discount_id );
 
-		if ( empty( $d ) ) {
-			continue;
+		// Retrieve total flat rate amount.
+		if ( 'flat' === $d->get_type() ) {
+			foreach ( $products as $product ) {
+				edd_get_item_discount_amount( $product, $products, array( $d ) );
```

Comment:
Should this be assigned to a variable, or be added to `$edd_flat_discount_total`? I'm not sure what's going on here.

Reply:
@robincornett The `global $edd_flat_discount_total;` inside `edd_get_item_discount_amount()` gets updated. I'll add some comments to clear this up.